packages/dropdown/__tests__/dropdown.spec.ts

```diff
@@ -8,6 +9,7 @@ const MOUSE_ENTER_EVENT = 'mouseenter'
 const MOUSE_LEAVE_EVENT = 'mouseleave'
 const CLICK = 'click'
 
+const _sleep = () => sleep(250)
```

Comment:
I think there is no need to wrap a private sleep function with default time again here. Guess just only short for  `sleep(250)`.

Reply:
Yes, sleep(250) has called multi times in the code below, and I think it's more concise if do so